challenge.rb

```diff
@@ -67,6 +96,33 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
-# Replace this comment with your code.
+def fizzbuzz_calc (min max)
+  list = {
+  'Fizz' => [],
+  'Buzz' => [],
+  'FizzBuzz' =>[],
+  'other' => []}
+
+  i = 1
+  while i <= max && i > max
+    if i%5 == 0 && i%3 == 0
+      puts "Fizzbuzz"
+      list['FizzBuzz'].push(i)
+    elsif i%3 == 0
+      puts 'Fizz'
+      list['Fizz'].push(i)
+    elsif i%5 == 0
+      puts 'Buzz'
+      list['Buzz'].push(i)
+    else
+      puts i
+      list['other'].push(i)
+    end
+    i += 1
```

Comment:
Technically, this solution might work (though it contains the unnecessary `list` object). However, we did also say that you should use the `fizz_buzz_calculator` method from Q4 as part of your solution.